code-challenges.js

```diff
@@ -39,26 +49,53 @@ var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var vowelTester2 = "throw"
 // Expected output: 3
 
+const firstVowel = (str) => {
+  let vowels = ["a", "e", "i", "o", "u"];
+  // Converts str into an array split by letters
+  let strArr = str.split("");
+  // Loops through each strArr element
+  for (let i = 0; i < strArr.length; i++) {
+    // If the vowels array includes the current value of strArr
+    // Return index of strArr[i]
+    if (vowels.includes(strArr[i])) {
+      return strArr.indexOf(strArr[i]);
+    }
+  }
+}
 
 
 
 
 
 // --------------------5) Create a function that takes three arguments - two numbers and a mathematical operation (+, -, *, /) and performs the given calculation. If the input tries to divide by 0, return: "Can't divide by 0!"
 
-
-
+const calculator = (num1, symbol, num2) => {
+  if (symbol == "/" && num2 == 0) {
```

Comment:
totally works. just curious about the use of `==` instead of `===`. i don't have a problem with it, as long as you can speak to why you made the choice. only being nit-picky because you _will_ be asked why you decided to do `==` in an interview.